### PR TITLE
fix(init): Reap child processes

### PR DESCRIPTION
**Pull request by bradbeam**
Should take care of our issue with Zombies

Signed-off-by: Brad Beam <brad.beam@talos-systems.com>

### CODE REVIEW

File: hack/containerd.toml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+root = "/var/lib/containerd"
+state = "/run/containerd"
+oom_score = 0
+subreaper = true
```
Comment (andrewrynhard) [inline hack/containerd.toml]: Think we should just have this?

Comment (andrewrynhard) [reply hack/containerd.toml]: I take it that flags take precedence over the config?

Comment (bradbeam) [reply hack/containerd.toml]: i believe so; the above is generated from `containerd config`

Comment (andrewrynhard) [reply hack/containerd.toml]: cool

**Pull request merged**